systray_menu_unix.go

```diff
@@ -224,6 +224,34 @@ func addSeparator(id uint32, parent uint32) {
 	refresh()
 }
 
+func hideSeparator(id uint32, parent uint32) {
+	instance.menuLock.Lock()
+	defer instance.menuLock.Unlock()
+	menu, _ := findLayout(int32(parent))
+	for _, i := range menu.V2 {
+		item := i.Value().(*menuLayout)
+		if item.V0 == int32(id) {
+			item.V1["visible"] = dbus.MakeVariant(false)
+			refresh()
+			return
+		}
+	}
+}
+
+func showSeparator(id uint32, parent uint32) {
+	instance.menuLock.Lock()
+	defer instance.menuLock.Unlock()
+	menu, _ := findLayout(int32(parent))
```

Comment:
Any reason for this to require `menuLock` to be acquired?

Reply:
This lock is used to protect maps inside the `menuLayout` object. It is used basically every time the `menuLayout` is changed. Do you think we can remove it here?